actionview/lib/action_view/base.rb

```diff
@@ -294,7 +294,7 @@ def in_rendering_context(options)
       old_view_renderer  = @view_renderer
       old_lookup_context = @lookup_context
 
-      if !lookup_context.html_fallback_for_js && options[:formats]
+      if options[:formats]
         formats = Array(options[:formats])
         if formats == [:js]
           formats << :html
```

Comment:
This looks like it might need to be ✂️  too.

Reply:
I thought that too, but I'm not 100% sure.  I *think* it might be required for templates that are children of JS templates. I'm good removing this too, but I don't know. cc @jhawthorn